scripts/perftune.py

```diff
@@ -614,14 +618,18 @@ def __dev_is_hw_iface(self, iface):
     def __check_dev_is_bond_iface(self):
         if not os.path.exists('/sys/class/net/bonding_masters'):
             return False
```

Comment:
This will be an issue.
You should be consistent with the type the method returns.
With your patch it is `dict{<NIC name> -> bool}`.